sourcegraph/sourcegraph
# Adjust spacing and sizes for different aggregation UI screens

Fixes https://github.com/sourcegraph/sourcegraph/issues/41428

## Background
This PR adjusts error, loading, no-data and query error UI screens for the sidebar mode
- Makes bar skeleton chart taller 
- Increase bar skeleton bars for the sidebar panel mode (in order to make it proportional for new sizes)
- Adjust height for bar chart on the full UI mode  

## Test plan
- Run `repo:^github\.com/sourcegraph/sourcegraph$ insights` and check loading state and data state
- Switch to files aggregation mode 
- Add `type:commit`
- Check error aggregation mode
- Switch to author aggregation mode and replace `type:commit` with `type:diff` (it should trigger Run aggregation screen)
- Check run aggregation screen 
- Check the same flow for the full UI mode



## App preview:

- [Web](https://sg-web-vk-fix-aggregation-ui-screens.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-hzelfyxwom.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
